lambda/src/main/scala/pricemigrationengine/handlers/EstimationHandler.scala

```diff
@@ -88,7 +88,7 @@ object EstimationHandler extends CohortHandler {
 
     lazy val earliestStartDateForAMonthlySub =
       for {
-        randomFactor <- Random.nextIntBetween(0, 3)
+        randomFactor <- Random.nextIntBetween(12, 15)
```

Comment:
Do we need a conditional here where we test whether the first payment date is less than 12 months ago? It will mean the first payment date needs to be included in the import.

Reply:
Addressed in the last section I have just added to the description

Reply:
Problem has been solved with further commits